app/views/results/_university.html.erb

```diff
@@ -0,0 +1,39 @@
+<% site_distance = @results_view.site_distance(course)%>
+
+<dt class="govuk-list--description__label">Location</dt>
+
+<dd>
+  <ul class="govuk-list govuk-!-margin-top-0 govuk-!-margin-bottom-0">
+    <li>
+      <% if course.further_education? %>
+        <span class="govuk-list--description__hint govuk-!-margin-top-0">University</span>
+      <% else %>
+        <span class="govuk-list--description__hint govuk-!-margin-top-0">Placement schools</span>
+        <details class="govuk-details" data-module="govuk-details">
+          <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+              <% if site_distance < 11 %>
+                Placement schools are near you
+              <% elsif site_distance < 21 %>
+                Placement schools might be near you
+              <% else %>
+                Placement schools might be in commuting distance
+              <% end %>
```

Comment:
Could this not be put into a decorator/helper? It would make it easier to test which result is returned based on the distance. it would also mean that we don't have to set ruby variables in the view.

Reply:
done